static/js/periodShortcuts.js

```diff
@@ -0,0 +1,44 @@
+var _ = require('ep_etherpad-lite/static/js/underscore');
+var utils = require('./utils');
+
+var DOT_KEY_CODE = 190;
+var SCENE_EXTENSIONS = ["EXT.", "INT.", "E/I.", "I/E."];
+
+exports.createSceneIfHasSceneExtension = function (rep, attributeManager, editorInfo) {
+  var lineNumber = rep.selStart[0];
+  var $line = getLineFromLineNumber(lineNumber)
+  var lineTextIsASceneExtension = textOfLineIsSceneExtension($line);
+  var lineIsAction = isLineAction($line);
+
+  if (lineTextIsASceneExtension && lineIsAction && hasNotSelection(rep)) {
+    changeLineToHeading(lineNumber, editorInfo, attributeManager);
+  }
+}
+
+exports.periodKeyWasPressed = function (evt) {
+  return evt.which === DOT_KEY_CODE && evt.type === 'keyup';
+}
+
+var getLineFromLineNumber = function (lineNumber) {
```

Comment:
essa função deve estar em algum utils.js por aí, tem chance de nem precisar dela se vc for mover pro outro plugin...